package.json

```diff
@@ -37,24 +37,25 @@
     "release:version": "./scripts/release/version.js"
   },
   "devDependencies": {
-    "@commitlint/cli": "^14.1.0",
-    "@commitlint/config-conventional": "^14.1.0",
-    "@rollup/plugin-replace": "^3.0.0",
-    "@rollup/plugin-typescript": "^8.2.5",
-    "@types/jest": "^27.0.1",
-    "@types/node": "^16.7.13",
-    "execa": "^5.1.1",
-    "husky": "^7.0.4",
+    "@commitlint/cli": "^17.0.2",
+    "@commitlint/config-conventional": "^17.0.2",
+    "@rollup/plugin-replace": "^4.0.0",
```

Comment:
Seems unlikely this would break us: https://github.com/rollup/plugins/blob/master/packages/replace/CHANGELOG.md#v400